Quickstart.md

```diff
@@ -0,0 +1,197 @@
+        ),
+        
+    )
+    cli = create_cli(configuration)
+    cli()
+
+# ------------------------------------------------------------------------------
+# ENTRYPOINT
+# ------------------------------------------------------------------------------
+
+if __name__ == '__main__':
+    main()
+```
+
+Make sure the myapp.py has executable permission. In order to do this, you need to cd into the directory of myapp.py in your terminal and execute the following command
+
+```
+chmod +x myapp.py
+```
+
+### Dockerfile Setup
+
+Outside of the src folder, create a docker file and paste in the following settings so we can assemble our image
+
+```Dockerfile
+# filename: Dockerfile
+
+FROM brightsparklabs/appcli
+
+ENTRYPOINT ["./myapp.py"]
+WORKDIR /app
+
+# install compose if using it as the orchestrator
+RUN pip install docker-compose
+
+COPY requirements.txt .
+RUN pip install --requirement requirements.txt
+COPY src .
+
+ARG APP_VERSION=latest
+ENV APP_VERSION=${APP_VERSION}
+```
+
+We can see in the dockerfile, it is referring to a requirements.txt file which we will need to create. Go ahead and create a requirements.txt file in the same location as your Dockerfile and enter in the following `bsl-appcli==1.3.4` 
+This will install the appcli package in the image.
+
+### Create a docker compose file
+You will need to create a docker compose file which will be placed in the baseline folder. Give it the default name docker-compose.yml
+
+We will be using an Echo-Server for our example application which is useful for testing purposes as it replicates the request sent by the client and sends it back. More information on this container can be found [here](https://ealenn.github.io/Echo-Server/pages/quick-start/docker.html#run)
+
+Paste in the following code into your docker compose file.
+
+```Dockerfile
+version: '3'
+services:
+  echo-server:
+    image: ealen/echo-server:0.5.1
+```
+
+
+### Directory check
+
+Before we build our application please make sure your directory reflects the following structure
+
+```
+├── src
+│   ├── resources
+│   │   ├── settings.yml
+│   │   ├── stack-settings.yml
+│   │   ├── templates
+|   |           ├── baseline
+|   |           |       ├── docker-compose.yml
+|   |           ├── configurable
+|   | 
+│   ├── myapp.py
+|
+├── requirements.txt
+├── Dockerfile    
+```
+
+
+### Build the container
+
+```
+docker build -t brightsparklabs/myapp --build-arg APP_VERSION=latest .
+```
+
+### View and Run the installed script
+
+While it is not mandatory to view the script before running, it is highly recommended.
+
+```
+docker run --rm brightsparklabs/myapp:latest install
+```
+
+Once you can successfully see the script printed on the command line, you are ready to install
+
+```
+docker run --rm brightsparklabs/myapp:latest install | sudo bash` 
+```
+
+### Intialise
+
+Once you've succesfully ran the install script you can initialise the app by typing in the following command
```

Comment:
Avoid contractions in tech writing. Use `Once you have`

typo in `successfully`